app/services/manage_action.rb

```diff
@@ -32,5 +33,9 @@ def queue_message
       }
     }
   end
+
+  def generate_referring_user
+    @params[:referring_user] = "/rest/v1/user/#{actionkit_user_id(@params.delete(:referring_akid))}/" if @params.has_key? :referring_akid
```

Comment:
hmm I guess this whole PR is about tight AK integration. still, this line could at least use some documentation, and might be nice a little less compacted